src/freenas/usr/local/lib/middlewared_truenas/alert/source/failover.py

```diff
@@ -193,14 +193,17 @@ def check_sync(self):
                 pass
             try:
                 if len(fobj['phrasedvolumes']) > 0:
-                    if not self.middleware.call_sync('failover.encryption_status'):
-                        alerts.append(Alert(NoFailoverEscrowedPassphraseAlertClass))
+                    keys = self.middleware.call_sync('failover.encryption_keys')['geli']
```

Comment:
I am not sure I follow, isn't this supposed to run on MASTER controller ? And if yes, we are going to get the alert on MASTER controller which is good ? If the user does a failover, this will run on the new node which will have become MASTER and we should be good there too ?

Reply:
I mis-read this originally. Looks good.